ansible/install_galaxy_roles.yml

```diff
@@ -38,7 +38,7 @@
       command: >-
         ansible-galaxy collection install
         -r "{{ requirements_path }}"
-        -p collections
+        -p {{ lookup('config', 'COLLECTIONS_PATHS')[0] | quote }}
```

Comment:
Is first collection paths always in user land?
No risk of being system path?

Reply:
Ah we saw concurrent colelction issues on admin recently. Ansible Workshops was installing into default `~/.ansible/collections` with 2 different versions. We saw lots of issues, race conditions. That has been resolved but a simple `ansible-galaxy install` without a `-p` should be avoided!